examples/dagger2/src/main/java/org/liquigraph/examples/dagger2/dao/SentenceRepository.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.examples.dagger2.dao;
+
+import org.liquigraph.examples.dagger2.entity.Sentence;
+
+import java.util.Optional;
+
+public interface SentenceRepository {
+
+    Optional<Sentence> findOne(String cypherQuery);
```

Comment:
actually: my previous signature proposal is a bit weird (sorry :sweat_smile:), I think
```java
Optional<Sentence> findOne()
```
would make more sense (no need to expose the query to the client)